e2e-tests/gef/cypress/integration/unissued-facility-change-from-preview-AIN.spec.js

```diff
@@ -0,0 +1,258 @@
+      aboutFacilityUnissued.coverEndDateYear().should('have.value', '');
+    });
+
+    it('update facilities later link should take you back to application preview', () => {
+      applicationPreview.unissuedFacilitiesReviewLink().click();
+      unissuedFacilityTable.updateIndividualFacilityButton(0).click();
+
+      aboutFacilityUnissued.issueDateDay().type(oneMonthDay);
+      aboutFacilityUnissued.issueDateMonth().type(oneMonthMonth);
+      aboutFacilityUnissued.issueDateYear().type(oneMonthYear);
+
+      aboutFacilityUnissued.shouldCoverStartOnSubmissionNo().click();
+      aboutFacilityUnissued.coverStartDateDay().type(twoMonthsDay);
+      aboutFacilityUnissued.coverStartDateMonth().type(twoMonthsMonth);
+      aboutFacilityUnissued.coverStartDateYear().type(twoMonthsYear);
+
+      aboutFacilityUnissued.coverEndDateDay().type(threeMonthsOneDayDay);
+      aboutFacilityUnissued.coverEndDateMonth().type(threeMonthsOneDayMonth);
+      aboutFacilityUnissued.coverEndDateYear().type(threeMonthsOneDayYear);
+      aboutFacilityUnissued.continueButton().click();
+
+      unissuedFacilityTable.successBanner().contains(`${unissuedFacilitiesArray[0].name} is updated`);
+      unissuedFacilityTable.rows().should('have.length', unissuedFacilitiesArray.length - 1);
+      unissuedFacilityTable.continueButton().should('not.exist');
+      unissuedFacilityTable.updateFacilitiesLater().click();
+    });
+
+    it('preview review facility stage has correct headers and shows 1 updated facilities', () => {
+      applicationPreview.reviewFacilityStage().contains('Review facility stage');
+
+      applicationPreview.updatedUnissuedFacilitiesHeader().contains('The following facility stages have been updated to issued:');
+      applicationPreview.updatedUnissuedFacilitiesList().contains(unissuedFacilitiesArray[0].name);
+      applicationPreview.unissuedFacilitiesReviewLink().should('not.exist');
+      applicationPreview.submitButtonPostApproval().should('exist');
+    });
+
+    it('facility table should have change links on the changed to issued facilities', () => {
+      // not be able to change 1 name, but can change to issued
+      applicationPreview.facilitySummaryListRowValue(0, 0).contains(MOCK_FACILITY_FOUR.name);
+      applicationPreview.facilitySummaryListRowAction(0, 0).should('have.value', '');
+
+      applicationPreview.facilitySummaryListRowAction(0, 1).should('have.value', '');
+      applicationPreview.facilitySummaryListRowValue(0, 2).contains('Unissued');
+      applicationPreview.facilitySummaryListRowAction(0, 2).contains('Change');
+
+      applicationPreview.facilitySummaryListRowKey(0, 2).should('not.have.value', 'Date issued to exporter');
+      applicationPreview.facilitySummaryListRowKey(0, 3).should('not.have.value', 'Cover start date');
+      applicationPreview.facilitySummaryListRowKey(0, 4).should('not.have.value', 'Cover end date');
+
+      // should not be able to change number 3 has previously issued
```

Comment:
Should this be this?

`// should not be able to change number 3 as it has already been issued`

Reply:
done